vmdb/app/models/host_vmware_esx.rb

```diff
@@ -23,7 +23,8 @@ def vim_reboot(force = false)
 
   def vim_enter_maintenance_mode(timeout = 0, evacuate_powered_off_vms = false)
     with_provider_object do |vim_host|
-      $log.info "MIQ(Host.vim_enter_maintenance_mode) Invoking with: timeout: [#{timeout}], evacuate_powered_off_vms: [#{evacuate_powered_off_vms}]"
+      $log.info "MIQ(Host.vim_enter_maintenance_mode) Invoking with:"\
+        " timeout: [#{timeout}], evacuate_powered_off_vms: [#{evacuate_powered_off_vms}]"
```

Comment:
I'd really not like the long lines broken up for this PR if possible...The long line rubocop is a suggestion not a mandate